plugins/modules/inventory_workflow_manager.py

```diff
@@ -1788,6 +1932,34 @@ def provisioned_wired_device(self):
 
         return self
 
+    # Added function
+    def assign_device_to_site(self, device_ip, assign_network_device_to_site):
+        try:
+            self.log(assign_network_device_to_site)
```

Comment:
self.log("Assigning device with IP {0} to site. Request data: {1}".format(device_ip, assign_network_device_to_site))

Reply:
updated the code